sentry-ruby/spec/sentry/net/http_spec.rb

```diff
@@ -9,6 +9,28 @@
     ::Logger.new(string_io)
   end
 
+  context "with IPv6 addresses" do
+    before do
+      perform_basic_setup
```

Comment:
Sorry that I forgot to mention with this approach you need to enable tracing here.

```suggestion
      perform_basic_setup do |config|
        config.traces_sample_rate = 1.0
      end
```